specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -4242,16 +4246,20 @@
 		"properties": {
 			"name": {
 				"type": "string",
-				"description": "The name of the output."
+				"description": "The name path of the output."
 			},
 			"targetName": {
 				"type": "string",
 				"description": "The target name of the output."
 			}
 		},
-		"required": ["name",
-		"targetName"],
-		"description": "output field mapping for a skill."
+		"required": [
+		    "name", "targetName"
```

Comment:
I believe targetName is optional and defaults to name

Reply:
Fixed